_posts/2017-12-03-hamburger-emoji.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title: Important news about the hamburger emoji
+date: 2017-12-03
+---
+
+For those of you reading my blog, I thought I will bring your attention to a very important piece of information: [Google fixed the burger emoji in Android 8.1](https://techcrunch.com/2017/11/28/whew-google-fixed-the-burger-emoji-in-android-8-1/). Yep, that's right, Android emoji used to have the cheese at the bottom, but they are "fixing" it:
+
+![Hamburger Right and Wrong]({{ "/assets/blog/google-burger.jpg" | absolute_url }}){:height="40%" width="40%"}
+
+_Image from [Tech Crunch](https://techcrunch.com/2017/11/28/whew-google-fixed-the-burger-emoji-in-android-8-1/)_
+
+<br/>
+I wonder how much work is involved for this change to go live. Creating a new image for the burger is easy but then what is the process of getting it released?
```

Comment:
Maybe _required_ instead of involved.  If you stick with 'involved' I think the sentence should read 'i wonder how much work is involved _in_ making this change go live'